src/features/embalm/stepContent/hooks/useCreateSarcophagus.ts

```diff
@@ -372,7 +395,7 @@ export function useCreateSarcophagus() {
     allowance,
     hasApproved,
     incrementStage,
-    processArchCommsException
+    processArchCommsException,
```

Comment:
I don't see any changes besides whitespace on this file. Hopefully I've not missed anything, but we need to unify our auto-formatters so they work off the same parameters. 

I thought we implemented this across projects earlier in development, but either something's changed since then, or I thought wrong.

Reply:
running `npm run prettier` prior to opening the PR should solve this issue -- I think everyone has auto-run-prettier on save in their IDEs, but could be for some reason certain rules aren't being applied.

Reply:
There is actually one change in this file: under `uploadAndSetDoubleEncryptedFile()` we are adding the `fileName` to the payload. 

And yeah, these eslint/prettier inconsistencies are getting annoying. We need to make sure we all have eslint/prettier set up properly.